### PR TITLE
docs(): add a code of conduct and info to README

### PR DESCRIPTION
**Pull request by TylorS**

### CODE REVIEW

**Comment by briancavalier** [comment]
Awesome, thanks @TylorS!  I should have time to dig into this later today.

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- Please be kind and courteous. There's no need to be mean or rude.
+- Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+- Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+- We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+- Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact one the Cycle.js team immediately. Whether you're a regular contributor or a newcomer, we care about making this community a safe place for you and we've got your back.
```
Comment (briancavalier) [inline CODE-OF-CONDUCT.md]: Cycle.js -> Most.js.  Is there a page we can link to that lists the Maintainers team members?

Comment (TylorS) [inline CODE-OF-CONDUCT.md]: :+1: I'll do this when I get back from my trip. I'm sure there is a link. But ya'll have to make your memberships public

**Comment by TylorS** [comment]
Fixed!

**Pull request merged**